### PR TITLE
NOS News, new extension.

### PR DESCRIPTION
**Pull request by Joran-Dob**
NOS News, new extension.

## Description

Added an new extension to display content from the biggest Dutch news site. Which can be found here: https://nos.nl/. Looking to improve this in future versions with more news categories than only the latest news.

## Type of change
- New extension

## Screencast

Screencast can be found here: [https://youtu.be/8RkD-vOKlpU](https://youtu.be/8RkD-vOKlpU)

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Comment by Joran-Dob** [comment]
Small note, I used Dutch for some properties as it made more sense to me as the content from the news website is also in Dutch. For example, the time when the article was added. 

If this needs to be changed I am happy to do so.

File: extensions/nos-news/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+function Actions(props: { item: Parser.Item }) {
+  const { push } = useNavigation();
+
+  return (
+    <ActionPanel title={props.item.title}>
+      <ActionPanel.Section>
+        <ActionPanel.Item
+          title="View item"
```
Comment (RSO) [inline extensions/nos-news/src/index.tsx:69]:
```suggestion
          title="View Item"
```

File: extensions/nos-news/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "name": "nos-nieuws",
+  "title": "NOS Nieuws",
+  "description": "Used to display the latest NOS nieuws from nos.nl",
+  "icon": "command-icon.png",
+  "author": "jorandob",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show NOS News",
```
Comment (RSO) [inline extensions/nos-news/package.json:12]: Personally I don't think that "Show" is needed here, but that might be a matter of taste.

```suggestion
      "title": "NOS News",
```

**Review by RSO — CHANGES_REQUESTED** [review]
Looks good to me! 1 small change request with regards to action panel naming, other than that I think this is good to go.

**Comment by Joran-Dob** [comment]
@RSO great! Thanks, I will just quickly update the naming on the action panel.

File: extensions/nos-news/package.json (hunk 2 above)
Comment (Joran-Dob) [reply extensions/nos-news/package.json:12]: I was thinking the show might be nice as I want to add more variations/sources later on. Something like 'Show Lastest News from NOS' or something similar. 😄

**Review by RSO — APPROVED** [review]

**Pull request merged**